### PR TITLE
Add SoftSV 1.4.2.

### PR DESCRIPTION
**Pull request by cbrueffer**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/softsv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - bamtools ==2.4.1
+    - boost ==1.62.0
```
Comment (epruesse) [inline recipes/softsv/meta.yaml]: Does it need this version, or just this version and up?

File: recipes/softsv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - makefile.diff
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - bamtools ==2.4.1
```
Comment (epruesse) [inline recipes/softsv/meta.yaml]: Pin to `=2.4.*` or `>2.4.1,<2.5.0a0`?

File: recipes/softsv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - boost ==1.62.0
+    - seqan ==2.1.1
```
Comment (epruesse) [inline recipes/softsv/meta.yaml]: as above

File: recipes/softsv/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+test:
+  commands:
+    - SoftSV --help
+
+about:
+  home: https://sourceforge.net/projects/softsv
+  license: GPLv3
```
Comment (epruesse) [inline recipes/softsv/meta.yaml]: Must have `license_file` when  license is GPL

File: recipes/softsv/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  summary: 'SoftSV is a tool for the detection of small and large deletions, inversions, tandem duplications and translocations from paired-end sequencing data.'
```
Comment (epruesse) [inline recipes/softsv/meta.yaml]: Try to keep this to one line. It's shown on `anaconda.org/bioconda/`*`packagename`* in a place where more than a line looks weird. It's the title:

```
summary: Find deletions, inversions and tandem duplications in paired end reads
description: |
  SoftSV is a tool for the detection of small and large deletions, inversions, tandem
  duplications and translocations from paired-end sequencing data.

  Lore ipsum
 
  more ipsum
```    

(While the description field is an entire pane at the bottom where you can put lots of googleable text)

**Comment by epruesse** [comment]
@cbrueffer This seems to be a problem with `conda_build.api.render` in the presence of pinnings in `conda_build_config`. You can only pin to specific version numbers, not using match-specs. So write `bamtools 2.4.1` if that is truly necessary.

**Comment by epruesse** [comment]
Retry this tomorrow. I'm guessing the CDN on the anaconda end isn't updating, or taking longer than usual.

**Comment by dpryan79** [comment]
@bioconda-bot please update

**Comment by cbrueffer** [comment]
Obsolete.

**Pull request closed without merging**